### ISSUE TITLE
Release 2.0.0

### ISSUE THREAD

**Issue by f2prateek** [square/tape#152]
Just wanted to make a list of what's left for 2.0.0.

* [ ] Update website and examples https://github.com/square/tape/pull/151
* [ ] Fix todos https://github.com/square/tape/search?utf8=%E2%9C%93&q=todo

Anything else?

**Comment by NightlyNexus** [comment]
Sounds good.

Also, wishlist:
Samples can always come in later.
If Converter artifacts end up being something people want, they could be introduced in a point release.

**Comment by darktiny** [comment]
Does it mean code in `master` branch is ready for production usage ?

**Comment by f2prateek** [comment]
I've been using it in a production app for a few months.
Generally the code on master is always ready to be used for production. Only the APIs aren't guaranteed to be stable (i.e. they could be changed/removed).

**Comment by NightlyNexus** [comment]
Should those exceptions be a new internal type, or is wording all that's needed? I think I've seen other code do a different type in this situation, but I can't recall where or why.

**Comment by sandeep-voxer** [comment]
Hello, I haven't seen any new activity here since June, and it's been almost 3 years since the last release i.e 1.2.3 - any idea when 2.0 will be released?

**Comment by JakeWharton** [comment]
No

**Comment by JakeWharton** [comment]
It's already very usable, so I encourage you to try it out

**Comment by sandeep-voxer** [comment]
Well I guess my concern is whether it's stable enough (in terms of bugs/crashes not API) to use for production apps.

**Comment by NightlyNexus** [comment]
Yes. Master is stable and tested. This code powers production apps and sdks.
The API is not stable but probably won't change much if at all.

**Comment by sandeep-voxer** [comment]
Got it, thank you!

**Comment by NightlyNexus** [comment]
For the remaining exceptions, Java 8's UncheckedIOException seems like a good choice in the future. A simple RuntimeException seems good for now.

**Comment by goghAta** [comment]
Yes, release it please.

**Comment by dimhold** [comment]
Please, release

**Comment by JakeWharton** [comment]
There is still no code to perform queue migration from v1 to v2 format.

**Comment by f2prateek** [comment]
@JakeWharton do you think that should be a blocker for 2.0.0-RC1?

**Comment by JakeWharton** [comment]
Yes, because it wouldn't be a candidate for release. Perhaps an alpha
instead.

**Comment by f2prateek** [comment]
Makes sense. I'll rework https://github.com/square/tape/pull/195 to be an alpha for now.

**Comment by sjoerd-michels** [comment]
Any idea when v2 will be released?
